uaclient/tests/test_config.py

```diff
@@ -306,6 +310,7 @@ def test_nonroot_attached(self, _m_getuid):
             'techSupportLevel': status.INAPPLICABLE,
         }
         assert expected == cfg.status()
+        # cfg.status is idempotent
```

Comment:
This addition seems more aspirational than functional. Did you accidentally miss a line? :-P

Reply:
This is showing up in my local branch per commitish 60744219dd54fc413cdb82b1c6cb1dccfee86d22
+        # cfg.status() idempotent
+        assert expected == cfg.status()